gulpfile.js

```diff
@@ -63,26 +70,35 @@ gulp.task('css', () =>
           'src/assets/stylesheets',
           'node_modules/govuk_frontend_toolkit/stylesheets',
           'node_modules/govuk-elements-sass/public/sass',
+          'node_modules/font-awesome/scss/',
         ],
       }))
-    .pipe(gulp.dest('dist/public/stylesheets/'))
+    .pipe(gulp.dest(`${stagePath}stylesheets/`))
+);
+
+gulp.task('clean', () =>
+  gulp.src('dist/public', { read: false, allowEmpty: true })
+    .pipe(clean())
 );
 
-gulp.task('revision:rename', ['js', 'css'], () =>
+gulp.task('revision:rename', () =>
   gulp.src([
-    'dist/public/**/*.html',
-    'dist/public/**/*.css',
-    'dist/public/**/*.js',
-    'dist/public/**/*.{jpg,png,jpeg,gif,svg}'])
+    `${stagePath}**/*.html`,
+    `${stagePath}**/*.css`,
+    `${stagePath}**/*.js`,
+    `${stagePath}**/*.{jpg,png,jpeg,gif,svg,otf,eot,ttf,woff,woff2}`])
     .pipe(debug())
     .pipe(rev())
-    .pipe(revDelOriginal())
```

Comment:
How comes we're removing this? Is it no longer required?

Reply:
I copied the changes made by @mangogrin from cya. When I reviewed that change in cya it seemed to work nicely